waku/waku_api/rest/admin/handlers.nim

```diff
@@ -111,6 +111,32 @@ proc installAdminV1PostPeersHandler(router: var RestRouter, node: WakuNode) =
 
     return RestApiResponse.ok()
 
+proc installAdminV1GetFilterSubsHandler(router: var RestRouter, node: WakuNode) =
+  router.api(MethodGet, ROUTE_ADMIN_V1_FILTER_SUBS) do () -> RestApiResponse:
+
+    if node.wakuFilter.isNil():
+      return RestApiResponse.badRequest("Error: Filter Protocol is not mounted to the node")
+
+    var subscriptions: seq[FilterSubscription] = @[]
+
+    if not node.wakuFilter.isNil():
```

Comment:
Can be remove since it's checked above

Reply:
Oh yes! Was supposed to remove it but for sure got distracted and forgot 😶

Will fix it now, thanks so much!